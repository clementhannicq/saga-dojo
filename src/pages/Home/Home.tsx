import React, { SFC } from "react";

const Home: SFC<{}> = () => <p>Homepage</p>;

export default Home as any;
