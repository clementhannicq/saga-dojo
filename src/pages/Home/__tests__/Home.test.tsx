import { shallow } from "enzyme";
import React from "react";
import Home from "../Home";

describe("Pages", () => {
  describe("Home", () => {
    it("should render", () => {
      const wrapper = shallow(<Home />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
