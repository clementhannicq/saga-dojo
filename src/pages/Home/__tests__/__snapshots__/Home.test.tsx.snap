// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages Home should render 1`] = `
<p>
  Homepage
</p>
`;
